specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-01-31/consumption.json

```diff
@@ -1132,6 +1266,87 @@
         }
       },
       "x-ms-azure-resource": true
+    },
+    "PriceSheetListResult": {
+      "description": "price sheet result. It contains the pricesheet associated with billing period",		
+      "properties": {
+        "value": {
+          "description": "Price sheet",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/PriceSheet"
+          }
+        },
+        "nextLink": {
+          "description": "The link (url) to the next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+	  "PriceSheet": {
```

Comment:
change this to pricesheetmodel to avoid this warning
{
  "type": "Warning",
  "code": "OperationIdNounConflictingModelNames",
  "message": "OperationId has a noun that conflicts with one of the model names in definitions section. The model name will be disambiguated to 'PriceSheetModel'. Consider using the plural form of 'PriceSheet' to avoid this. Note: If you have already shipped an SDK on top of this spec, fixing this warning may introduce a breaking change.",
  "id": "R2063",
  "validationCategory": "SDKViolation",
  "providerNamespace": "Microsoft.Consumption",
  "resourceType": "pricesheets",
  "sources": [
    "file:///C:/g/AI/git/Hao/azure-rest-api-specs/specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-01-31/consumption.json:502:3 ($.paths[\"/{scope}/providers/Microsoft.Consumption/pricesheets/default\"].get.operationId)"
  ],
  "jsonref": "file:///C:/g/AI/git/Hao/azure-rest-api-specs/specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-01-31/consumption.json:502:3 ($.paths[\"/{scope}/providers/Microsoft.Consumption/pricesheets/default\"].get.operationId)",
  "json-path": "file:///C:/g/AI/git/Hao/azure-rest-api-specs/specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-01-31/consumption.json:502:3 ($.paths[\"/{scope}/providers/Microsoft.Consumption/pricesheets/default\"].get.operationId)"
}